src/main/java/seedu/address/model/Model.java

```diff
@@ -248,5 +248,5 @@
      */
     void updateFilteredCartIngredientList(Predicate<Ingredient> predicate);
 
-    void removeAllCartIngredient();
+    void removeAllCartIngredients();
```

Comment:
I just went to take a look at AB3, the clear command is executed by using `model.setAddressBook(new AddressBook())`. I think we could do the same for clearing the cart rather than adding this new method, or otherwise we should add this method for the cookbook and inventory too. I didn't notice it in the original PR implementing the cart clear command.

Reply:
alright, i'll revert to using `set`